### ISSUE TITLE
[HOLD #42118] Status not updated in Newdot for reports approved in classic

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#54642]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.79-2
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @quinthar 
**Slack conversation** (hyperlinked to channel name): [expense](https://expensify.slack.com/archives/C06ML6X0W9L/p1735353035623299)

## Action Performed:
1. Submit an expense to approver
2. Login as approver and approve the report in classic
3. As submitter observe the report

## Expected Result:
Report action message and status updated in real time
## Actual Result:
Status not updated and displayed as `Waiting for you to approve the expense's`, Report action message displayed as Approved
Status updated after page refresh
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

![image (3)](https://github.com/user-attachments/assets/2ffd68a1-f58d-4a25-b4c3-6307c14e1ad5)

![Snip - (2) New Expensify - Google Chrome](https://github.com/user-attachments/assets/4f4f68e7-3ae1-4e52-9f9f-8f3f5677683a)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by jliexpensify** [comment]
Alex B has a theory [here](https://expensify.slack.com/archives/C06ML6X0W9L/p1735499295486279?thread_ts=1735353035.623299&cid=C06ML6X0W9L). Asing if it should be an `Internal` issue.

**Comment by jliexpensify** [comment]
Waiting for someone to pick it up

**Comment by jliexpensify** [comment]
Internal issue, waiting

**Comment by jliexpensify** [comment]
Still waiting

**Comment by jliexpensify** [comment]
Not overdue, internal issue waiting to be picked up

**Comment by jliexpensify** [comment]
Not overdue

**Comment by jliexpensify** [comment]
Internal, waiting

**Comment by jliexpensify** [comment]
Still waiting for an internal engineer

**Comment by jliexpensify** [comment]
Not overdue, this is waiting on an Internal Engineer

**Comment by jliexpensify** [comment]
Confirmed with Vit that this is on hold until https://github.com/Expensify/App/issues/42118 is completed.

**Comment by jliexpensify** [comment]
On hold

**Comment by jliexpensify** [comment]
Still on hold

**Comment by jliexpensify** [comment]
On hold

**Comment by jliexpensify** [comment]
On hold

**Comment by jliexpensify** [comment]
Still on hold

**Comment by mallenexpensify** [comment]
Holdin'
